projects/nature40-app/src/app/app.component.ts

```diff
@@ -315,4 +317,40 @@ export class AppComponent implements OnInit, AfterViewInit {
         );
     }
 
+    /**
+     * This handler stores the user viewport extent on leaving the app and retrieves it the next time.
+     * The data is stored in local storage, s.th. it is not reflected using multiple browsers.
+     * If there is no extent stored it uses the *Uniwald* extent as a starting point.
+     */
+    private setupInitialZoom() {
+        const PATH_PREFIX = window.location.pathname.replace(/\//g, '_').replace(/-/g, '_');
+        const storage_key = `${PATH_PREFIX}_zoom_extent`;
+
+        let stored_extent: [number, number, number, number] = JSON.parse(localStorage.getItem(storage_key));
+
+        if (!stored_extent) {
+            stored_extent = [8.66, 50.82, 8.69, 50.84]; // default to Uniwald
```

Comment:
while this works for this project, can we load it from the config file? (later)

Reply:
d63ce39